packages/client/mutations/AddMissingJiraFieldMutation.ts

```diff
@@ -0,0 +1,35 @@
+import graphql from 'babel-plugin-relay/macro'
+import {commitMutation} from 'react-relay'
+import {StandardMutation} from '../types/relayMutations'
+import {AddMissingJiraFieldMutation as TAddMissingJiraFieldMutation} from '../__generated__/AddMissingJiraFieldMutation.graphql'
+
+graphql`
+  fragment AddMissingJiraFieldMutation_part on AddMissingJiraFieldSuccess {
```

Comment:
A pattern we use in the code is to suffix the name of a fragment with the name of the it's resolved field. So in this case it should be `AddMissingJiraFieldMutation_dimensionField`

Reply:
Oh, I see. Will fix it.

Reply:
that's true for queries! for mutations, the suffix is the name of the subscription channel (that way it's easy to spot if we change the channel the subscription is published on: https://github.com/ParabolInc/parabol/blob/0d9a3bde5900aedd8fb49c7437f8694e88b486c1/packages/client/subscriptions/MeetingSubscription.ts)